### ISSUE TITLE
Document Neuter Filter

### ISSUE THREAD

**Issue by lifeinafolder** [wycats/rake-pipeline-web-filters#45]
It would be nice to have some sort of documentation on what Neuter Filter is and how it works. The only piece of text I could find about it is this link: http://blog.nulayer.com/post/23294069876/building-simple-javascript-libraries-with-rake-pipeline

**Comment by wagenet** [comment]
Agreed. The original point of NeuterFilter is to take minispade requires and to order them into a single file with no requires.
